### DISCUSSION TITLE
Creating repositories for future workshops

### DISCUSSION THREAD

**Discussion by samarthswarup** [MABSworkshop/MABSworkshop.github.io#1] [Announcements]
As we create repositories for future MABS workshops, I suggest we follow these steps:

* Always have the main repository, [MABSworkshop.github.io](https://github.com/MABSworkshop/MABSworkshop.github.io), be the repository for the latest iteration of the workshop.
* Once the workshop is done, fork it with the name mabs\<year\>, e.g., mabs2024, to archive it. Set its URL in the About section to be mabsworkshop.github.io/mabs\<year\>.
* Update the main repository for the next year's workshop.

Thoughts/comments?

**Comment by koenvandam** [comment]
Sounds perfect, thanks for suggesting this Samarth!

**Comment by gnardin** [comment]
@samarthswarup 

I agree with these steps. I would just suggest to include some specific sub-items under the last bullet:

- Update the main repository for the next year's workshop.
  + Update the **About** content to reflect the updated year and edition of the workshop
  + Remove the files and update the README file under the `articles` folder
  + Remove the files under the `presentations` folder
  + Update the `_config.yml`, the `index.markdown`, and the `*.md` files in the root folder

Thank you,

**Comment by lorigf** [comment]
I also agree this sounds good. 

What might be nice to have is a menu/link to the previous years' repos from the main site. Currently, it seems you can only reach the previous editions through Jaime's website (https://www2.pcs.usp.br/~mabs/) or via Google.

Speaking of which, Jaime asked us to inform him when a new website is online so we can add it to the website of the series that he is hosting. 🙂 (and update the link of the 2024 edition)

/Fabian
